src/components/SearchPageComponents/SidePanel/SidePanel.js

```diff
@@ -0,0 +1,91 @@
+import { Chip, Stack, Box, Typography } from "@mui/material";
+import React from "react";
+import { useState, useEffect } from "react";
+import axios from "axios";
+const SidePanel = ({ searchText }) => {
+  const [list, setList] = useState([]);
+  const [listAdv, setListAdv] = useState([]);
+
+  useEffect(() => {
```

Comment:
you can create the fetcData function outside the useEffect , and use one function instead of the two fetchData functions , and pass it through the paratameters the API link

Reply:
done